python/pyarrow/_parquet.pyx

```diff
@@ -879,6 +879,15 @@ cdef encoding_name_from_enum(ParquetEncoding encoding_):
         ParquetEncoding_BYTE_STREAM_SPLIT: 'BYTE_STREAM_SPLIT',
     }.get(encoding_, 'UNKNOWN')
 
+cdef encoding_enum_from_name(str encoding_name):
```

Comment:
Small code style nit: can you leave two blank lines between functions/defintions? so one more above and below this object (you will see that the other definitions here do the same)